src/source/Common/Util.c

```diff
@@ -18,6 +18,7 @@ STATUS convertTimestampToEpoch(PCHAR expirationTimestampStr, UINT64 nowTime, PUI
     MEMSET(&ioTExpiration, 0x00, SIZEOF(struct tm));
 
     // iot expiration timestamp format "YYYY-MM-DDTHH:mm:SSZ"
+    CHK(STRLEN(expirationTimestampStr) != STRLEN("YYYY-MM-DDTHH:mm:SSZ"), STATUS_TIMESTAMP_STRING_UNRECOGNIZED_FORMAT);
```

Comment:
Recommend using STRNLEN for expirationTimestampStr and an ARRAY_SIZE for the pre-defined string. The format should be in the private header

Reply:
Also, perhaps rename the error message name to reflect the length name check

Reply:
Can we define this format: "YYYY-MM-DDTHH:mm:SSZ" instead of using the string as is?